p2p/interface.go

```diff
@@ -0,0 +1,13 @@
+package p2p
+
+import "github.com/bloxapp/ssv-spec/types"
+
+// Broadcaster is the interface used to abstract message broadcasting
+type Broadcaster interface {
+	Broadcast(message types.Encoder) error
```

Comment:
this means that the wrapping into ssv message needs to be done in the network implementation? 
I think it's ambiguous which type of msg are we expecting here? 
Maybe just enforce SSVMessage and that's it?

Reply:
Agreed, changed in https://github.com/bloxapp/ssv-spec/pull/87/commits/7e61689470710bc09f2ec5a108363c6c7ec467c4